src/excmds.ts

```diff
@@ -2740,15 +2743,20 @@ export async function winopen(...args: string[]) {
         switch (args[0]) {
             case "-private":
                 createData.incognito = true
-                args = args.slice(1, args.length)
+                args.shift()
                 firefoxArgs = "--private-window"
```

Comment:
If `-private` and `-c` are given, we should throw an error. We don't want people to accidentally open things they think are private in a normal container. : )

Reply:
Agree, I made it fail explicitly.